test/cases/api.sh

```diff
@@ -516,6 +523,9 @@ function Test_verifyComposeResultGCP() {
 
   echo "${SSH_USER}:$(cat "$GCP_SSH_KEY".pub)" > "$GCP_SSH_METADATA_FILE"
 
+  # add "gitlab-ci-test" label to the image
+  # $GCP_CMD compute images add-labels "$GCP_IMAGE_NAME" --labels=gitlab-ci-test=true
+  
```

Comment:
redundant comment?

Reply:
We need this line to add the label to the image. I'm doing some debugging, as for some reason, that line gives me the next error: 
https://gitlab.com/osbuild/ci/image-builder/-/jobs/2117306772#L2971

Reply:
Ah IC, yea this image is made by the stage deployment, and as such isn't in the cockpituous account. It's located in the account used for the service, these get automatically cleaned up after 14 days.

Reply:
Ohh, that's why. Thanks!!